### PR TITLE
Added item classes, amended schema, updated readme, bumped version

### PR DESCRIPTION
**Pull request by guywillis**
Enabled classes for each tabs item for both framework and authoring tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,50 +22,52 @@ The attributes listed below are used in *components.json* to configure **Tabs**,
...
+**tabTitle** headings will be replaced by **...** if the heading is too long for the container.
 
 ----------------------------
-**Version number:**  2.0.4  
-**Framework versions:** 2.0+  
-**Author / maintainer:** Kineo  
-**Accessibility support:** WAI AA   
-**RTL support:** Yes  
-**Cross-platform coverage:** To be confirmed  
+**Version number:**  2.0.4
```
Comment (tomgreenfield) [inline README.md]: * Lost line breaks
* Version `2.1`

File: bower.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "adapt-tabs",
     "framework": ">=2",
-    "version": "2.0.4",
+    "version": "2.0.5",
```
Comment (tomgreenfield) [inline bower.json]: `2.1.0`.

**Review by tomgreenfield — COMMENTED** [review]
Please also update the example JSON.

File: README.md [hunk 3]
```diff
@@ -22,50 +22,52 @@ The attributes listed below are used in *components.json* to configure **Tabs**,
...
+**_items** (array): Multiple items may be created. Each *_item* represents one element of the tabs component and contains values for **tabTitle**, **title**, **body**, and **_graphic**.
 
 >**tabTitle** (string): This text is displayed in the actual tab heading. Recommend keeping this title short.
 
 >**title** (string): This text is displayed as the content panel's header. It is displayed when the tab has been selected.
 
->**body** (string): This content will be displayed when the learner opens this content panel. It may contain HTML.  
+>**body** (string): This content will be displayed when the learner opens this content panel. It may contain HTML.
 
 >**_graphic** (object): An optional image which is displayed below the item body when the learner opens this content panel. It contains values for *src* and *alt*.
```
Comment (moloko) [inline README.md]: Change last sentence to "It contains valus for src, alt and _classes"

Comment (guywillis) [reply README.md]: The classes are applied to the item level rather than the graphic level

Comment (moloko) [reply README.md]: in that case might be better to move the entry up so it's between body and _graphic?

Comment (guywillis) [reply README.md]: Makes sense, I'll move it.

File: bower.json (hunk 2 above)
Comment (guywillis) [reply bower.json]: Done

File: README.md (hunk 1 above)
Comment (guywillis) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (guywillis) [reply README.md]: Done

**Review by tomgreenfield — APPROVED** [review]

**Pull request merged**